pages/2_subscriptions_report.py

```diff
@@ -53,11 +146,124 @@
     with row1_col1:
         st.markdown("**Number of New Subscriptions**")
 
+        # Number of New Subscriptions
+        new_subscriptions_data = subscription_data.groupby('subscription_started_month').size()
```

Comment:
Since we are using the `subscription_started_month` as the group by field here I noticed that the date range filter then doesn't apply as expected to these charts (since the date range filter is using the `created_at` field). Do you think it would be possible to look for new subscriptions which have occurred within the date range period? Or is that not possible given the fake data we have and we instead need to adjust the fake data to tell a more accurate story?

Reply:
Notice how the date range is just for a portion of a single month. However, the New Subscriptions and New Subscriptions by Plan charts have nearly a years worth of data.


![image](https://github.com/user-attachments/assets/c3d75079-c9d2-40fa-aac1-af98754a082a)

Reply:
I have updated all the charts for their respective dates to fall in the date range.